app/controllers/drivers_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DriversController < ApplicationController
+  def index
+    @drivers = Driver.all
```

Comment:
Try to use helper methods instead of instance variables to pass data from controller to view.
```
helper_method :drivers
...

private
def drivers
  @drivers ||= Driver.all
end
```

So in view you can use `drivers` instead of `@drivers`

Reply:
interesting i dont see this often, but its a decent idea

Reply:
how do i 

> Try to use helper methods instead of instance variables to pass data from controller to view.
> 
> ```
> helper_method :drivers
> ...
> 
> private
> def drivers
>   @drivers ||= Driver.all
> end
> ```
> 
> So in view you can use `drivers` instead of `@drivers`

how do I implement this?

Reply:
ok never mind i got it